api/pkg/transformer/feast/feature_retriever.go

```diff
@@ -95,6 +92,7 @@ type Options struct {
 	BatchSize               int           `envconfig:"FEAST_BATCH_SIZE" default:"50"`
 	CacheEnabled            bool          `envconfig:"FEAST_CACHE_ENABLED" default:"true"`
 	CacheTTL                time.Duration `envconfig:"FEAST_CACHE_TTL" default:"60s"`
+	CacheSizeInMB           int           `envconfig:"CACHE_SIZE_IN_MB" default:"100"`
```

Comment:
I prefer `FEAST_CACHE_SIZE_IN_MB` as env name

Reply:
It will be a breaking change if we rename the env var.